src/com/dotmarketing/portlets/browser/ajax/BrowserAjax.java

```diff
@@ -1143,24 +1177,24 @@ public boolean moveFile ( String inode, String folder ) throws Exception {
 
         try {
             Identifier ident=APILocator.getIdentifierAPI().findFromInode(inode);
-            IHTMLPage page = ident.getAssetType().equals("htmlpage")
+            IHTMLPage page = ident.getAssetType().equals("htmlpage") 
```

Comment:
Usually use the constant in the left side is safer than in the right right, for instance consider the following sentence:

String s = null;

if (s.equals("something)) { // will throw NPE

but if you do:
if ("something".equals(s)) { // will return false, with is good and safer